internal/infrastructure/db/models/boil_queries.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by SQLBoiler 4.14.1 (https://github.com/volatiletech/sqlboiler). DO NOT EDIT.
```

Comment:
Can you get your SQLBoiler and protoc up-to-date? Good to not go backwards if we don't have to.